File: dark/release/dark_code/compiler/NIM_CODE/modules/dark_math.nim
import math
import random


proc dark_sqrt*(x: float): float =
    return math.sqrt(x)

proc dark_pow*(x: float, y: float): float =
    return math.pow(x, y)

proc dark_floor*(x: float): float =
    return math.floor(x)

proc dark_ceil*(x: float): float =
    return math.ceil(x)

proc dark_pi*(): float =
    return math.PI

proc dark_random*(a: int, b: int): int =
    random.randomize()
    return random.rand(a..b)

File: dark/dark_code/compiler/py_dark_code/dark_math.py
import math as python_math
import random as python_random


def sqrt(x):
    """Вычисляет квадратный корень из числа."""
    return python_math.sqrt(x)

def pow(x, y):
    """Вычисляет базовую величину в степени exp."""
    return python_math.pow(x, y)

def floor(x):
    """Возвращает пол номера."""
    return python_math.floor(x)

def ceil(x):
    """Возвращает максимальное значение числа."""
    return python_math.ceil(x)

def pi():
    """Возвращает значение числа PI."""
    return python_math.pi

def random(a, b):
    """Возвращает случайное значение с плавающей точкой в диапазоне от a до b."""
    return python_random.randint(a, b)

def randint(a, b):
    """Возвращает случайное целое число в диапазоне от a до b."""
    return python_random.randint(a, b)